### ISSUE TITLE
TextOptions.VerticalAlignment is aligning text lower than it used to

### ISSUE THREAD

**Issue by kevinwell** [SixLabors/ImageSharp.Drawing#91]
### Prerequisites

- [x] I have written a descriptive issue title
- [x] I have verified that I am running the latest version of ImageSharp.Drawing
- [x] I have verified if the problem exist in both `DEBUG` and `RELEASE` mode
- [x] I have searched [open](https://github.com/SixLabors/ImageSharp.Drawing/issues) and [closed](https://github.com/SixLabors/ImageSharp.Drawing/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported

### Description
On the left is the result from version 1.0.0-beta0007 and on the right is version 1.0.0-beta0010
![old](https://user-images.githubusercontent.com/41668407/94395755-b69d6f00-01a3-11eb-9dd1-3ca60143fd86.jpeg) ![new](https://user-images.githubusercontent.com/41668407/94395788-c6b54e80-01a3-11eb-973f-4d43e9b9bc66.jpeg)

### Steps to Reproduce
```
var center = new PointF(image.Width / 2f, image.Height / 2f);

image.Mutate(i => i
    .DrawText(new TextGraphicsOptions
    {
        TextOptions = new TextOptions{
            HorizontalAlignment = HorizontalAlignment.Center,
            VerticalAlignment = VerticalAlignment.Center,
            }
    }, "TEST", font, Color.White, center));
```

### System Configuration

- ImageSharp.Drawing version: 1.0.0-beta0010
- Other ImageSharp packages and versions: ImageSharp 1.0.1
- Environment (Operating system, version and so on): Windows 10 Version 2004
- .NET Framework version: .NET Core 3.1
- Additional information:

**Comment by JimBobSquarePants** [comment]
This appears to be related to https://github.com/SixLabors/Fonts/issues/161

**Comment by JimBobSquarePants** [comment]
Fixed with SixLabors/Fonts#161
